rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml

```diff
@@ -183,3 +183,75 @@ spec:
             team: pipelines
             # link will be available once https://issues.redhat.com/browse/SRVKP-5899 is complete
             runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/slos/pac-performance.md
+        - alert: PipelinesControllerPipelineRunRecLeasesNotDistributed
+          expr: count by (source_cluster) (count by (lease_holder, source_cluster) (kube_lease_owner{lease=~"tekton-pipelines-controller.github.com.tektoncd.pipeline.pkg.reconciler.pipelinerun.reconciler..*", lease_holder!=""})) <= 1
```

Comment:
I think we should have test cases with more than one `source_cluster`, especially if we aggregate by this label.

Reply:
Operators precedence here is a little challenging. What comes first? the `<=` or the count? Maybe we can add parentheses to make the statement clearer.

Reply:
IMO this thread is resolved now.